Cargo.toml

```diff
@@ -63,4 +63,4 @@ protobuf = { git = "https://github.com/pingcap/rust-protobuf", rev = "82b49fea7e
 protobuf-codegen = { git = "https://github.com/pingcap/rust-protobuf", rev = "82b49fea7e696fd647b5aca0a6c6ec944eab3189" }
 
 [workspace]
-members = [ "stress" ]
+members = ["stress", "ctl"]
```

Comment:
add a newline

Reply:
> add a newline
You mean we should add a empty line below the line members?

Reply:
yes, see https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
💯